docs/user/example-cases/e3sm-cases/harvey-flooding/e3sm_harvey.md

```diff
@@ -0,0 +1,91 @@
+# Overview
+
+The E3SM-RDycore model has been tested on Perlmutter and Frontier for the RDycore's 5-day Hurricane Harvey benchmark. The E3SM-RDycore simulation uses a data-land configuration with an active river model. In an E3SM-RDycore run, RDycore can run on CPUs and GPUs. The overall workflow for an E3SM-RDycore run is as follows:
+
+- Get the code:
+    - Clone the E3SM fork from [https://github.com/rdycore/e3sm](https://github.com/rdycore/e3sm).
+    - Switch to the E3SM-RDycore development branch and initialize submodules of E3SM **and** RDycore.
+```bash
+git clone git@github.com:rdycore/e3sm
+cd e3sm
+git checkout rdycore/mosart-rdycore/8b4c2d5df3-2024-04-05
+git submodule update --init
+cd externals/rdycore
+git submodule update --init
+```
+
+- Create, build, and run a case
+    1. Compile RDycore to generate libraries (i.e. `librdycore.a`, `librdycore_f90.a`, `libcyaml.a`, `libyaml.a`, and `libcmocka.a`)
+    2. Create a E3SM case. Presently, the coupled model has been tested for a case with `--comspet RMOSGPCC --res MOS_USRDAT`.
```

Comment:
This should read "Create an E3SM case." Also, let's use "currently" instead of "presently." Presently actually means "very soon".